### ISSUE TITLE
`pat-contentbrowser` accessibiltiy improvements

### ISSUE THREAD

**Issue by krissik** [plone/mockup#1482]
A user tested ContentBrowserSelect-Widget with a screenreader and reported the following:

* Close and Home button are missing an accessible name, e.g. `aria-label`
* It is hard to figure out which column shows the contents of which folder b/c  the selected folder of the left column is just a background (based on css class `inPath`). The information which folder is select is not accessible.
